.github/workflows/footprint-tracking.yml

```diff
@@ -109,7 +109,9 @@ jobs:
           ELASTICSEARCH_KEY: ${{ secrets.ELASTICSEARCH_KEY }}
           ELASTICSEARCH_SERVER: "https://elasticsearch.zephyrproject.io:443"
           ELASTICSEARCH_INDEX: ${{ vars.FOOTPRINT_TRACKING_INDEX }}
+        shell: bash
```

Comment:
It would be good to just set the default shell to bash at the job level:

```
    defaults:
      run:
        shell: bash
```

See https://github.com/zephyrproject-rtos/sdk-ng/blob/main/.github/workflows/ci.yml#L431-L433 for example.

Reply:
yeah, changed (assuming it doesn't affect negatively other job steps here).
I wonder is there a need in `--entrypoint` option for the docker container when github action has its own way to select which shell to run ?
https://github.com/zephyrproject-rtos/zephyr/pull/80239/files#diff-b063339944aadb532da381f9121fa0058d199d3326798fe7588909876ad8af6dR30

Reply:
> I wonder is there a need in `--entrypoint` option for the docker container when github action has its own way to select which shell to run ?

Yeah, I do not recall why that was added; but, it probably is no longer necessary. Another thing to fix for the upcoming Docker image overhaul ...